docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+
+FROM alpine:latest AS builder
+
+ARG BUILD_TYPE
+ARG RELEASE
```

Comment:
Would you mind provide some instructions for the general users on how to build/run it? I mean a concise step-by-step Readme.md would suffice.

Reply:
I'll work on it